resources/quality/goofoo/tpu/goofoo_far_0.40_tpu_87a_standard.inst.cfg

```diff
@@ -0,0 +1,17 @@
+[general]
+version = 4
+name = Standard Quality
+definition = goofoo_far
+
+[metadata]
+setting_version = 16
+type = quality
+quality_type = normal
+material = goofoo_tpu_87a
+variant = 0.5mm Nozzle
```

Comment:
The name of the file is inconsistent with the variant used. Please rename the file to `resources/quality/goofoo/tpu/goofoo_far_0.50_tpu_87a_standard.inst.cfg`.